libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/BetaLabel.kt

```diff
@@ -36,7 +38,7 @@ fun BetaLabel(
                 shape = shape,
             )
             .padding(horizontal = 8.dp, vertical = 4.dp),
-        text = "BETA",
+        text = stringResource(CommonStrings.common_beta).uppercase(),
```

Comment:
Should we use the locale here? `string.uppercase(locale)` I mean.

Reply:
Maybe but I think it's safe to use `uppercase()`. We have other occurrences in the codebase so we can decide that later.